services/post_service.py

```
@@ -0,0 +1,8 @@
+from typing import Annotated
+
+from repositories.user_repository import UserRepository
+
+
+class PostService:
+    def __init__(self,user_repository: UserRepository):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)